src/merkleDistributor/model/Distribution.ts

```diff
@@ -0,0 +1,23 @@
+export type DistributionRecipients = {
+  [address: string]: Pick<DistributionRecipient, "amount" | "accountIndex" | "metadata">;
+};
+export type DistributionRecipientsWithProofs = { [address: string]: DistributionRecipient };
+
+export type DistributionRecipient = {
+  amount: string;
+  accountIndex: number;
+  windowIndex: number;
+  proofs: string[];
```

Comment:
Isn't this actually just a single `proof`?

Reply:
it is an array of strings. Should I rename this to `proof`?